README.md

```diff
@@ -24,8 +24,12 @@ an old beggar, X?", where X is the person passed into the method.
 Demonstrate calling the method with an argument of "young prince".
 
 Write your code here:
-```ruby
-# code here
+```
+def offerRose person
+  puts "Would you take this rose and help out an old beggar, #{person}?"
+end
+
+person "young prince"
```

Comment:
Ruby should be `snake_case` instead of  camelCase. Also, it would be `offer_rose "young prince" `and not `person "young prince"`